package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "metal-state-validators",
+  "version": "1.0.0",
+  "description": "Utility class with helpers for creating validators for Metal.js state",
+  "main": "lib/type.js",
+  "scripts": {
+    "compile": "babel --presets es2015 -d lib/ src/",
+    "prepublish": "npm run compile",
+    "test": "mocha --compilers js:babel-core/register ./test/**/*.js"
```

Comment:
I'll take the chance to answer your question about formatting standards here.

We do have standards for all the metal repos, and the main standard is to keep the same infrastructure for all of them, so that they follow the same directory structure and use the same scripts for building/testing/linting/formatting. That way you can always interact with any of the repos in the same way.

So, if you check the package.json of the other repos you'll notice that all of them use `gulp-metal`, and the test script comes from one of the gulp tasks it provides. For example: https://github.com/metal/metal-debounce/blob/master/package.json.

Also, there's always the same `.jshintrc` file, which you can copy from metal-debounce or any of the others.

Btw, we usually use [generator-metal](https://github.com/metal/generator-metal) to create new metal repos, so that the structure follows the expected pattern. That would already have created the right files and imported the default dependencies for you. So next time you can use it :)

After updating this to use `gulp-metal` you'll be able to run `gulp lint` and `gulp format`, which will cover most of our formatting guidelines. You'll also be able to run `gulp test:coverage` to check if the code coverage is good (we aim for 100%, since that's not hard to do and helps catch at least the most common cases). If you run `gulp test:coverage:open` instead the report will open in your browser so you can see covered lines.